### PR TITLE
MAINT-26916: Add custom checks to the add/edit group form.

### PR DESCRIPTION
**Pull request by mdiha**
* I added a check on text length of group Name field in order to display a warning message if the length is lesser than 3 characters or greater than 30 characters.
* I added a check on group Name field in order to display a warning message if it starts with a number.
* I added a check on text length of group Label field in order to display a warning message if the length is lesser than 3 characters or greater than 50 characters.
* I added a check on text length of group Description field in order to display a warning message if the length is greater than 255 characters.

### CODE REVIEW

**Review by ahamdi — CHANGES_REQUESTED** [review]
Add description to the PR

**Comment by mdiha** [comment]
> Add description to the PR

Okay

**Comment by ahamdi** [comment]
Will this block the saving of the group ?

**Comment by mdiha** [comment]
> Will this block the saving of the group ?

yes it does till the label length is between 3 and 50 characters

File: webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue [hunk 1]
```diff
@@ -161,20 +161,64 @@ export default {
       this.drawer = true;
     },
     saveGroup(event) {
+      const startWithRegex =  /^\d/;
       const regex =  /^[a-zA-Z0-9-_]+$/;
```
Comment (ahamdi) [inline webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue]: Could you combine those two regex in one ?

Comment (mdiha) [reply webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue:165]: First one is for: Name starts with number: "The "Group Name" field must start with a character and must not contain special characters."
Second one is for: special characters.
And each one is used apart.

Comment (mdiha) [reply webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue]: Done

**Review by ahamdi — APPROVED** [review]

**Pull request merged**